app/config/auth.js

```diff
@@ -1,33 +1,46 @@
 const Joi = require('joi')
+const { getEnvironmentVariable } = require('../lib/environment-helpers')
+
+const SECOND = 1000
+const MINUTE = 60 * SECOND
+const HOUR = 60 * MINUTE
 
 // Define config schema
 const schema = Joi.object({
   enabled: Joi.boolean().default(false),
-  azure: Joi.object({
-    clientSecret: Joi.string().allow(''),
+  oidc: Joi.object({
+    privateKey: Joi.string().allow(''),
+    identityVerificationPublicKey: Joi.string().allow(''),
     clientId: Joi.string().allow(''),
-    authority: Joi.string().allow('')
+    redirectUri: Joi.string().allow(''),
+    postLogoutUri: Joi.string().allow(''),
+    discoveryEndpoint: Joi.string().allow('')
   }),
   cookie: Joi.object({
     password: Joi.string().required(),
-    ttl: Joi.number().default(60 * 60 * 1000)
+    ttl: Joi.number().default(HOUR + MINUTE)
```

Comment:
Should be '*' not '+'

Reply:
the calculations are done on lines 4 - 6:

- SECOND = 1000
- MINUTE = 60 * SECOND = 60 * 1000
- HOUR = 60 * MINUTE = 60 * 60 * 1000
- FINAL RESULT = HOUR + MINUTE = (60 * 60 * 1000) + (60 * 1000)

Reply:
Yes, sorry, missed the fact you were going for 61 mins. Originally it was 60 mins but I think I tweaked it recently